pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -58,10 +58,6 @@ const (
 	// desiredStateOfWorldPopulatorLoopSleepPeriod is the amount of time the
 	// DesiredStateOfWorldPopulator loop waits between successive executions
 	desiredStateOfWorldPopulatorLoopSleepPeriod time.Duration = 1 * time.Minute
-
-	// reconcilerSyncDuration is the amount of time the reconciler sync states loop
-	// wait between successive executions
-	reconcilerSyncDuration time.Duration = 5 * time.Second
```

Comment:
The number of PV should not matter since the syncState() issues requests per node not per volume. If your AWS cluster has a large number of nodes or has multiple zones of clusters, it will cause the problem. Since this PR makes it configurable, user can set it up depending on their cluster setup. What is the API call rate limit? 

Suppose it is set to 1 minute, since there are two API calls in for each node, the call rate will be Nx2x60 per hour